components/web/apps/org.wso2.carbon.apimgt.admin.app/src/main/fragments/policy-add/public/js/index.js

```diff
@@ -49,6 +55,7 @@ var addPolicy = function () {
         "id": 0,
         "enabled": false,
         "description": "",
+        "roleList" : "Internal/everyone",
```

Comment:
Still the back en support is no there. So for now we have hard coded this role. I think we can remove this